01-intro.Rmd

```diff
@@ -5,14 +5,35 @@ ottrpal::set_knitr_image_path()
 
 # Introduction
 
+This book contains a collection of vignettes that guide users through different ways to access data on the AnVIL platform. 
 
 ## Motivation
 
+One of the most common questions people have when they switch to using the AnVIL platform is "how do I get data onto it?" We created this book to provide small, step-by-step examples for users to try so that they can feel confident when moving data for their own research onto AnVIL. Each vignette is quick and designed to take ~15 minutes.
+
+It can be very exciting to learn how much data is at your fingertips! Once you have settled on some data to use, you'll want to bring it into AnVIL if it's not already there.
+
+Navigate to the menu on the left to get started!
 
 ## Target Audience  
 
-The course is intended for ...
+The course is intended for new and experienced users of the AnVIL platform. In order to follow the vignettes, **you must have an AnVIL account that is connected to a billing project.** Each vignette will incur only minimal costs.
```

Comment:
Might want to add something like "please see XXX if you don't have an AnVIL account yet."